bumble/device.py

```diff
@@ -4727,7 +4727,7 @@ async def create_cis(
         self, cis_acl_pairs: Sequence[tuple[int, Connection]]
     ) -> list[CisLink]:
         for cis_handle, acl_connection in cis_acl_pairs:
-            cis_id, cig_id = self._pending_cis.pop(cis_handle)
+            cis_id, cig_id = self._pending_cis.get(cis_handle, None)
```

Comment:
```suggestion
            cis_id, cig_id = self._pending_cis.get[cis_handle]
```

The returned type of dict[K, V] is considered V | None, while pop(K) returns V and raises KeyError if key isn't present, so it's more like dict[K]

Reply:
You can use `mypy bumble` to check before pushing.

Reply:
blindly accepted suggestion :-)

using `mypy bumble` I do get some errors in other modules:

```
bumble/data_types.py:414: error: Signature of "from_bytes" incompatible with supertype "builtins.int"  [override]
bumble/data_types.py:414: note:      Superclass:
bumble/data_types.py:414: note:          @classmethod
bumble/data_types.py:414: note:          def from_bytes(cls, bytes: Iterable[SupportsIndex] | SupportsBytes | Buffer, byteorder: Literal['little', 'big'] = ..., *, signed: bool = ...) -> SecurityManagerOutOfBandFlag
bumble/data_types.py:414: note:      Subclass:
bumble/data_types.py:414: note:          @classmethod
bumble/data_types.py:414: note:          def from_bytes(cls, data: bytes) -> SecurityManagerOutOfBandFlag
bumble/keys.py:259: error: Skipping analyzing "appdirs": module is installed, but missing library stubs or py.typed marker  [import-untyped]
bumble/device.py:2530: error: Incompatible types in assignment (expression has type "Host", variable has type "None")  [assignment]
bumble/device.py:2684: error: Incompatible types in assignment (expression has type "KeyStore", variable has type "None")  [assignment]
bumble/device.py:3559: error: Incompatible types in assignment (expression has type "bytes", variable has type "None")  [assignment]
bumble/device.py:3566: error: Argument "extended_inquiry_response" to "HCI_Write_Extended_Inquiry_Response_Command" has incompatible type "None"; expected "int"  [arg-type]
bumble/device.py:3716: error: Incompatible types in assignment (expression has type "OwnAddressType", variable has type "None")  [assignment]
bumble/pandora/utils.py:24: error: Library stubs not installed for "google.protobuf.message"  [import-untyped]
bumble/pandora/security.py:24: error: Library stubs not installed for "google.protobuf"  [import-untyped]
bumble/pandora/security.py:27: error: Cannot find implementation or library stub for module named "pandora.host_pb2"  [import-not-found]
bumble/pandora/security.py:28: error: Cannot find implementation or library stub for module named "pandora.security_grpc_aio"  [import-not-found]
bumble/pandora/security.py:29: error: Cannot find implementation or library stub for module named "pandora.security_pb2"  [import-not-found]
bumble/pandora/l2cap.py:25: error: Library stubs not installed for "google.protobuf"  [import-untyped]
bumble/pandora/l2cap.py:26: error: Cannot find implementation or library stub for module named "pandora.l2cap_grpc_aio"  [import-not-found]
bumble/pandora/l2cap.py:27: error: Cannot find implementation or library stub for module named "pandora.l2cap_pb2"  [import-not-found]
bumble/pandora/host.py:24: error: Library stubs not installed for "google.protobuf"  [import-untyped]
bumble/pandora/host.py:24: note: Hint: "python3 -m pip install types-protobuf"
bumble/pandora/host.py:24: note: (or run "mypy --install-types" to install all missing stub packages)
bumble/pandora/host.py:26: error: Cannot find implementation or library stub for module named "pandora"  [import-not-found]
bumble/pandora/host.py:27: error: Cannot find implementation or library stub for module named "pandora.host_grpc_aio"  [import-not-found]
bumble/pandora/host.py:28: error: Cannot find implementation or library stub for module named "pandora.host_pb2"  [import-not-found]
bumble/pandora/__init__.py:26: error: Cannot find implementation or library stub for module named "pandora.host_grpc_aio"  [import-not-found]
bumble/pandora/__init__.py:26: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
bumble/pandora/__init__.py:27: error: Cannot find implementation or library stub for module named "pandora.l2cap_grpc_aio"  [import-not-found]
bumble/pandora/__init__.py:28: error: Cannot find implementation or library stub for module named "pandora.security_grpc_aio"  [import-not-found]

```